        
byte Mode
byte DirR
byte DirL
uint16 SpeedR
uint16 SpeedL
